lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -1768,6 +1768,65 @@ defmodule Kernel.WarningTest do
     purge(TestMod)
   end
 
+  test "deprecate remote nullary zero-arity calls without parens" do
+    assert capture_err(fn ->
+             Code.eval_string("""
+             defmodule TestMod do
+                def foo(), do: nil
+             end
+
+             TestMod.foo
+             """)
+           end) =~
+             "missing parenthesis on call to TestMod.foo/0. parenthesis are always required on function calls without arguments"
```

Comment:
Let's update this message on the source to:

```suggestion
             "missing parentheses on call to TestMod.foo/0. Parentheses are always required on function calls without arguments"
```

In particular, parenthesis is singular, parentheses is plural, so we need to update accordingly.